### PR TITLE
Conditional JS inclusion in PA

### PR DESCRIPTION
**Pull request by koonsies**
## What's this PR do?
Change base-react to conditionally include the core CSS files

### CODE REVIEW

File: templates/base-react.html [hunk 1]
```diff
@@ -39,11 +39,13 @@
...
-        {% endcompress %}
+        {% block core-js %}
+            {% compress js %}
+                <script type="text/javascript" src="{% static "utils.184-4.js" %}"></script>
+                <script type="text/javascript" src="{% static "custom.166-06.js" %}"></script>
+                <script type="text/javascript" src="{% static "svg4everybody.min.js" %}"></script>
+            {% endcompress %}
+        {% endblock %}
         <script>
             svg4everybody();
```
Comment (aljordan82) [inline templates/base-react.html]: Should we add the script tag inside the block core-js?
Also, I am not sure if we need it in PA.

Comment (koonsies) [inline templates/base-react.html]: The svg4everybody(); call? Do you mean so that it can be overwritten by PA?

Comment (aljordan82) [inline templates/base-react.html]: so, if we don't include it in PA, it won't undefined.
I don't see why we need svg4everybody.js in PA

Comment (koonsies) [inline templates/base-react.html]: Good call.

One second.

**Pull request merged**